CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+### Introduction
```

Comment:
nitpicky comment - there are a bunch of markdown issues/warnings in this document that can be addressed. please use https://dlaa.me/markdownlint/ to see what the markdown issues/warnings are